### ISSUE TITLE
__version__ not up to date ?

### ISSUE THREAD

**Issue by touilleMan** [mongomock/mongomock#219]
Hi there,

Working to support mongomock into my [sync/async ODM](https://github.com/Scille/umongo) I realised the **version** of the mongomock package is not up to date:

``` shell
$ pip install -U mongomock==3.3.0
$ python -c "import mongomock; print(mongomock.__version__)"
3.0.0
```

So I'm not sure if it's something wanted (mongomock 3.3.0 aims at mocking pymongo 3.0.0) or a little oversight ;-)

**Comment by mdomke** [comment]
You're right. The `__version__` attribute is currently hardcoded. I'll fix that!

**Comment by pcorpet** [comment]
This was fixed in 4376165 thanks to an idea of @vmalloc
